### ISSUE TITLE
cities map not working with `plotly`

### ISSUE THREAD

**Issue by GitHunter0** [ipeaGIT/geobr#279]
Hi folks, for some reason the cities map is not working with `plotly` (while others like regions map are working flawlessly). Any idea what is happening? 
```
library(plotly)
library(geobr)
map_regions <- geobr::read_region(year=2010) |> sf::st_cast("MULTIPOLYGON")
map_cities <- geobr::read_municipality(code_muni="all", year=2010) |> sf::st_cast("MULTIPOLYGON")
```
While this works:
```
plot_ly() %>% add_sf(data = map_regions) 
```
This returns ```Error in `[.data.frame`(x, i) : undefined columns selected```:
```
plot_ly() %>% add_sf(data = map_cities) 
```

```
R version 4.1.1 (2021-08-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    
system code page: 65001

attached base packages:
[1] stats     graphics  grDevices utils    
[5] datasets  methods   base     

other attached packages:
[1] geobr_1.6.6   plotly_4.10.0 ggplot2_3.3.5
```

Thank you

**Comment by rafapereirabr** [comment]
Hi @GitHunter0 . Thanks for opening this issue. There could be some kind of conflict with `data.table`, but I'm not sure why.

```
library(plotly)
library(magrittr)
library(sf)
library(geobr)

# note the object has multiple classes
map_cities <- geobr::read_municipality(code_muni="all", year=2010)
class(map_cities)
> [1] "sf"         "data.table" "data.frame"

```
If if try `plot_ly()`, we get the error you pointed out.
```
plot_ly() %>% add_sf(data = map_cities) 

> Error in `[.data.frame`(x, i) : undefined columns selected

```

Now, when we remove the `data.table` class, then `plot_ly()` works.

```
map_cities <- as.data.frame(map_cities) %>% st_sf()
class(map_cities)
> [1] "sf"         "data.frame"

plot_ly() %>% add_sf(data = map_cities) 

```

**Comment by rafapereirabr** [comment]
Fixed in dev version with #45b99a20.
